core/testing/src/main/java/io/github/droidkaigi/confsched2023/testing/robot/TimetableScreenRobot.kt

```diff
@@ -58,6 +59,9 @@ class TimetableScreenRobot @Inject constructor(
                 )
             }
         }
+        // Timetable tab set initial tab with current date.
+        // To get the consistent test result, fix selected timetable tab to Day1 here.
+        clickTimetableTab(DroidKaigi2023Day.Day1.day)
         waitUntilIdle()
```

Comment:
`waitUntilIdle()` is also called in `clickTimetableTab`.
But I've kept it this way for clarity.